bin/cloudformation.py

```diff
@@ -93,20 +93,26 @@ def create_help(header, options):
                         choices = config_names,
                         metavar = "config_name",
                         help="Configuration to act upon (imported from configs/)")
-
+    parser.add_argument("--internal",
+                        action = "store_true",
+                        help="Attemps to execute cloudformation script without any credentials. Meant to use from internal aws instances")
     args = parser.parse_args()
 
-    if args.aws_credentials is None:
-        parser.print_usage()
-        print("Error: AWS credentials not provided and AWS_CREDENTIALS is not defined")
-        sys.exit(1)
+    if args.internal:
+        session = aws.use_iam_role()
 
+    else:
+        if args.aws_credentials is None:
```

Comment:
This would force us to not have AWS_CREDENTIALS as an env var or a file with credentials. Are we ok with that? I thought the internal flag would be more explicit? Can change if we want.

Reply:
I don't follow. Right now (in integration) the workflow is you can specify a file on the command line or using an environmental variable and if either are given an error is printed. My comment was to change that so that instead of printing an error it just assumed that you wanted to try the boto3 defaults.

When refactored this will all be removed as we will be loading credentials using boto3 and not explicitly as we have been doing.

Reply:
Done, way cleaner.